protocol/constants.go

```diff
@@ -0,0 +1,46 @@
+package protocol
+
+// Version is a type of supported versions of the protocol used in the accept header
+type Version string
+
+const (
+	// Iden3CommVersion1 is a V1 version of the protocol used in the accept header
+	Iden3CommVersion1 Version = "iden3comm/v1"
+)
+
+// AuthCircuits is a type of accepted authentication circuits
+type AuthCircuits string
+
+const (
+	// AuthCircuitsAuthV2 is authV2 accepted circuit
+	AuthCircuitsAuthV2 AuthCircuits = "authV2"
+	// AuthCircuitsAuthV3 is authV3 accepted circuit
+	AuthCircuitsAuthV3 AuthCircuits = "authV3"
```

Comment:
I think we need to remove AuthCircuitsAuthV3 as it doesn't exist yet and can never be created. Also maybe reuse seeing with CircuitId.AuthV2